var/spack/repos/builtin/packages/openfst/package.py

```diff
@@ -44,6 +44,13 @@ class Openfst(AutotoolsPackage):
     version('1.5.0',  'a24fee5ffe28744c6fb7b1a49e0006c4')
     version('1.4.1',  'ca8f1730b9b9b281e515611fa9ae23c0')
     version('1.4.0',  '662367ec91084ffab48ee9b5716de39c')
+    version('1.4.1-patch',  'ca8f1730b9b9b281e515611fa9ae23c0',
+            url='http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-1.4.1.tar.gz')
```

Comment:
Can you put these versions in order? I assume 1.4.1-patch is newer than 1.4.1 an older than 1.5.0.